build.gradle

```diff
@@ -106,6 +108,7 @@ dependencies {
         exclude module: 'annotations' // conflict with org.jetbrains:annotations, integTestRunner fails with error "codebase property already set"
     }
     compile "com.sun.mail:javax.mail:1.6.2"
+    compile 'com.google.code.gson:gson:2.8.6'
```

Comment:
If used only in tests, can it be moved to testCompile? Curious why XContentBuilder is not chosen which is already available.